src/components/SuggestionsProvider/SuggestionsList.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+import styled from '@stream-io/styled-components';
+import { CommandsItem, MentionsItem } from '../AutoCompleteInput';
+
+const Wrapper = styled.TouchableOpacity`
+  height: ${({ height }) => height};
+  position: absolute;
+  width: 100%;
+  z-index: 90;
+  ${({ theme }) => theme.messageInput.suggestions.wrapper.css}
+`;
+
+const Container = styled.View`
+  background-color: white;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  bottom: 10px;
+  height: ${({ theme, length }) =>
+    Math.min(
+      length * theme.messageInput.suggestions.container.itemHeight,
+      theme.messageInput.suggestions.container.maxHeight,
+    )};
+  margin-left: ${({ marginLeft }) => marginLeft};
+  position: absolute;
+  shadow-color: #000;
+  shadow-offset: 0px -3px;
+  shadow-opacity: 0.05;
+  width: ${({ width }) => width};
+  z-index: 100;
```

Comment:
alright well let's make sure we have something here that works on android and iOS as we know z-index has issues